config/Eclipse Code Style.epf

```diff
@@ -0,0 +1,328 @@
+#Fri Mar 09 22:05:27 CET 2018
```

Comment:
Is this file needed?

Reply:
Yes, as a reference file. And this file can still be imported manually. 
Mabye someone more experienced in gradle/scripting can create a simple script to copy the things from that file to the workspace org.eclipse.jdt.ui.prefs file